connector/src/main/scala/quasar/qscript/MapFunc.scala

```diff
@@ -221,10 +221,21 @@ object MapFunc {
   }
 
   def normalize[T[_[_]]: BirecursiveT: OrderT: ShowT, A: Show]
-      : CoEnv[A, MapFunc[T, ?], FreeMapA[T, A]] => CoEnv[A, MapFunc[T, ?], FreeMapA[T, A]] =
+      : CoMapFuncR[T, A] => CoMapFuncR[T, A] =
     (repeatedly(rewrite[T, A]) _) ⋘
       orOriginal(foldConstant[T, A].apply(_) ∘ (const => rollMF[T, A](Constant(const))))
 
+  def replaceJoinSides[T[_[_]]: BirecursiveT](left: Symbol, right: Symbol)
```

Comment:
This is where we replace the `JoinSideName` we previously encoded as a `StrLit`.